packages/perseus/src/components/math-input.jsx

```diff
@@ -1,174 +1,50 @@
-/* eslint-disable react/sort-comp */
 // @flow
 import classNames from "classnames";
 import $ from "jquery";
 import MathQuill from "mathquill";
-import PropTypes from "prop-types";
 import * as React from "react";
 import ReactDOM from "react-dom";
 import _ from "underscore";
 
 import TexButtons from "./tex-buttons.jsx";
 
-const PT = PropTypes;
+import type {ButtonSetsType} from "./tex-buttons.jsx";
+
+type ButtonsVisibleType = "always" | "never" | "focused";
+
+type Props = {|
+    className?: string,
+    value: string,
+    onChange: $FlowFixMe,
+    convertDotToTimes: boolean,
+    buttonsVisible: ButtonsVisibleType,
+    buttonSets: ButtonSetsType,
+    labelText?: string,
+    onFocus?: () => void,
+    onBlur?: () => void,
+|};
+
+type DefaultProps = {|
+    value: string,
+    convertDotToTimes: boolean,
+    buttonsVisible: ButtonsVisibleType,
```

Comment:
The pattern we use for default props is to use [indexed access types](https://flow.org/en/docs/types/indexed-access/).

```suggestion
    value: Props["values"],
    convertDotToTimes: Props["convertDotToTimes"],
    buttonsVisible: Props["buttonsVisible"],
```